### PR TITLE
Hotfix/storage used

### PR DESCRIPTION
**Pull request by rfornea**
StorageUsed (which has now been changed to StorageUsedInBytes) becomes out of sync when the user tries to upload lots of files.  Working on fixing that.

### CODE REVIEW

File: models/accounts.go [hunk 1]
```diff
@@ -186,13 +187,46 @@ func (account *Account) UseStorageSpaceInByte(planToUsedInByte int) error {
...
+	if err := tx.Model(&accountFromDB).Update("storage_used_in_byte",
+		gorm.Expr("storage_used_in_byte + ?", planToUsedInByte)).Error; err != nil {
+		tx.Rollback()
+		return err
+	} else {
+		if err := tx.Where("account_id = ?", account.AccountID).First(&accountFromDB).Error; err != nil {
+			tx.Rollback()
+			return err
+		} else {
+			if accountFromDB.StorageUsedInByte < int64(0) {
```
Comment (pzhao5) [inline models/accounts.go]: You should check storageUsedInByte < float64(accountFromDB.StorageLimit)
Since you can over commit it

Comment (rfornea) [reply models/accounts.go]: Okay I'll update it.

File: models/accounts.go [hunk 2]
```diff
@@ -186,13 +187,46 @@ func (account *Account) UseStorageSpaceInByte(planToUsedInByte int) error {
...
 		return errors.New("Unable to store more data")
 	}
-	account.StorageUsed = account.StorageUsed + inGb
 
-	return DB.Model(&account).Update("storage_used", account.StorageUsed).Error
+	if err := tx.Model(&accountFromDB).Update("storage_used_in_byte",
+		gorm.Expr("storage_used_in_byte + ?", planToUsedInByte)).Error; err != nil {
+		tx.Rollback()
+		return err
+	} else {
```
Comment (pzhao5) [inline models/accounts.go]: nit: you don't need this else statement

File: models/accounts.go [hunk 3]
```diff
@@ -186,13 +187,46 @@ func (account *Account) UseStorageSpaceInByte(planToUsedInByte int) error {
...
+		if err := tx.Where("account_id = ?", account.AccountID).First(&accountFromDB).Error; err != nil {
+			tx.Rollback()
+			return err
+		} else {
```
Comment (pzhao5) [inline models/accounts.go]: same here. remove it will make it read better

File: models/accounts.go (hunk 2 above)
Comment (rfornea) [reply models/accounts.go]: updated

File: models/accounts.go (hunk 3 above)
Comment (rfornea) [reply models/accounts.go]: updated

**Pull request merged**